pkgs/development/web/bun/default.nix

```
@@ -0,0 +1,19 @@
+{ stdenvNoCC, callPackage, lib }:
+let
+  version = "0.1.1";
+  pname = "bun";
+  meta = with lib; {
+    homepage = "https://bun.sh";
+    changelog = "https://github.com/Jarred-Sumner/bun/releases/tag/bun-${version}";
+    description = "Incredibly fast JavaScript runtime, bundler, transpiler and package manager – all in one.";
```

```suggestion
    description = "Incredibly fast JavaScript runtime, bundler, transpiler and package manager – all in one";
```

violation of CONTRIBUTING